### ISSUE TITLE
Configurable "power" of Spectrogram inside MelSpectrogram

### ISSUE THREAD

**Issue by tbright17** [pytorch/audio#624]
## 🚀 Feature
Right now the Spectrogram inside MelSpectrogram uses power 2 by default and this can't be modified when instantiating MelSpectrogram. Can we make it configurable as an argument of MelSpectrogram?

## Motivation
Some implementations of MelSpectrogram use power 1 (for example the VGGish code) while some use power 2. It's better to make it configurable when replicating others work.

## Pitch

https://github.com/pytorch/audio/blob/00d38203e401b8d9472a8f8394a10e2c309be02c/torchaudio/transforms.py#L408-L411

Here, can we make both "power" and "normalized" configurable rather than fixing them?

## Additional context

VGGish's melspectrogram implementation
https://github.com/tensorflow/models/blob/0fc994b6762b7bfe971cc9f370b40d2cd987fb66/research/audioset/vggish/mel_features.py#L92

**Comment by vincentqb** [comment]
Good idea. Do you want to open a PR to expose the power parameter with a default value of 2?

**Comment by tbright17** [comment]
> Good idea! Do you want to open a PR to expose the power parameter with a default value of 2?

Glad to do that!
